README.md

```diff
@@ -519,7 +611,7 @@ sudo docker run -d \
     simplelogin/app:3.2.2
 ```
 
-Next run the `email handler`
+`email handler` container:
```

Comment:
Why do we need to remove the `Next run the` here :)?

Reply:
i changed because you should add the 2 containers at the same time.

Reply:
fixed in https://github.com/simple-login/app/pull/309/commits/324640fbae1ddc6d3a2a8ace89e7935065555d8e